From bed6fa010029ba13e06a26a777b428a8192dbf8d Mon Sep 17 00:00:00 2001
From: tstegema <tstegema@90b65887-3827-0410-9a23-83215b262276>
Date: Tue, 9 Jul 2013 17:07:30 +0000
Subject: [PATCH 0438/1077] massai_systools.lib V1.0.0.6 Mantis 0027919:
 appControl: desktop can not be closed; ctfmon.exe must be terminated;

git-svn-id: svn://localhost/SelfServiceCommon/trunk@533 90b65887-3827-0410-9a23-83215b262276
---
 SelfServiceCommon/Massai/cpp/Systools/src/local_def.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/SelfServiceCommon/Massai/cpp/Systools/src/local_def.cpp b/SelfServiceCommon/Massai/cpp/Systools/src/local_def.cpp
index 52794f5a..dd97a490 100644
--- a/SelfServiceCommon/Massai/cpp/Systools/src/local_def.cpp
+++ b/SelfServiceCommon/Massai/cpp/Systools/src/local_def.cpp
@@ -5,7 +5,7 @@ VERSION_ID_DT(systools, 1, 0, 0, 6);
 /*!
 \page massai_systools.lib, massai_systools.lib - Release Notes
 
-\date 09.07..2013
+\date 09.07.2013
 \li V1.0.0.6
 \li Mantis 0027919: appControl: desktop can not be closed; ctfmon.exe must be terminated;
 mSysProcess::child added method to get parent process id: std::string getParentProcessId() const;
-- 
2.41.0.windows.1

